keycache/keycache.go

```diff
@@ -38,7 +48,7 @@ type ActiveUser struct {
 }
 
 type Cache struct {
```

Comment:
`Cache` should also have a comment, even though it didn't previously --- now's a good time to add one.